src/Tooltip2.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {Children, useEffect, useRef} from 'react'
+import Box from './Box'
+import {BetterSystemStyleObject, merge, SxProp} from './sx'
+import {useId} from './hooks/useId'
+import {isFocusable} from '@primer/behaviors/utils'
```

Comment:
@joshblack I found a little gem 🎀  This seems to be working as expected but let me know what you think!